src/views/v2/tickets/item/extra/available/quantity.php

```diff
@@ -18,7 +18,8 @@
  * @var Tribe__Tickets__Ticket_Object $ticket  The Ticket Object.
  * @var int $threshold                         The threshold value to show or hide quantity available.
  */
-if ( 0 !== $threshold && $threshold < $ticket->available() ) {
+
+ if ( 0 !== $threshold && $threshold < $ticket->available() ) {
```

Comment:
* :warning: Found precision alignment of 1 spaces.
* :no_entry_sign: Line indented incorrectly; expected 0 tabs, found 0

<sub>hash: 0146f5d7c24e65dcdbcca4e1918774e3</sub>